packages/context/src/context.ts

```diff
@@ -76,6 +74,11 @@ export class Context extends EventEmitter {
    */
   protected configResolver: ConfigurationResolver;
 
+  /**
+   * A debug function which can be overridden by subclasses
+   */
```

Comment:
Can you please expand the tsdoc comment with an example to show how to use this new property in practice? E.g.

```ts
import debugFactory from 'debug';
const debug = debugFactory('loopback:core:application');

class Application extends Context {
  constructor(parent: Context) {
    super(parent);
    this._debug = debug;
  }
}
```